src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
Yeah, I remember it was pretty easy for Babel 6, but dirtier for Babel 5, but perhaps worth including both in this PR? Thanks!